### PR TITLE
Adding support for meta at wt serve

### PR DESCRIPTION
**Pull request by jcenturion**
Adding:

- `wt serve -m key=value`
- `wt serve --metaFile myfile.json`

- [x] Update `webtask-runtime` once [https://github.com/auth0/webtask-runtime/pull/8](https://github.com/auth0/webtask-runtime/pull/8) is merged

### CODE REVIEW

**Comment by NotMyself** [comment]
I pulled this down and was using it yesterday. The create modifications work, but `--metaFile` support for **serve** does not work.

**Comment by jcenturion** [comment]
@NotMyself try again, `webtask-runtime` was not supporting meta. It is now.

File: package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
     "url": "https://github.com/ggoodman",
     "twitter": "filearts"
   },
-  "version": "6.5.2",
+  "version": "6.5.3",
```
Comment (ggoodman) [inline package.json]: Can we bump the minor? New feature.

**Comment by NotMyself** [comment]
`--meta-file` on `serve` is now pulling the file in but the `context.meta` object appears to be malformed.

Example:

screen shot 2017-10-04 at 9 38 29 am

File: package.json (hunk 1 above)
Comment (jcenturion) [reply package.json]: 👍

**Comment by NotMyself** [comment]
worksonmymachine

**Comment by jcenturion** [comment]
@NotMyself 👍

**Pull request merged**